File: MODUI/src/app/Models/payment.ts
export class Payment {
    public  paymentId : string
        public  amount : number
       
        public  userId : string
        
        public  mentorId : string
        
        public trainingId : string
        public  amountToMentor :number;

        constructor(){
            this.paymentId="P" + Math.round((Math.random()*9000));

        }
}

File: MODBackend/TrainingService/Models/Payment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace TrainingService.Models
{
    [Table("Paymentdtl")]
    public class Payment
    {
        [Key]
        public string PaymentId { get; set; }
        public int amount { get; set; }
        [ForeignKey("Userdtl")]
        public string UserId { get; set; }
        [ForeignKey("MentorDtl")]
        public string MentorId { get; set; }
        [ForeignKey("Training")]
        public string TrainingId { get; set; }
        public int AmountToMentor { get; set; }

        public Userdtl Userdtl { get; set; }

        public MentorDtl MentorDtl { get; set; }

        public Training Training { get; set; }
    }
}
